src/vast_tracker.js

```diff
@@ -327,6 +355,10 @@ export class VASTTracker extends EventEmitter {
    */
   setSkipDelay(duration) {
     if (!util.isValidTimeValue(duration)) {
+      this.emit('TRACKER-error', {
+        message: 'setSkipDelay parameter does not have a valid value',
+        parameter: duration,
```

Comment:
I don't find having multiple property for the tracker-error convenient, because it error prone and then to use we multiply the properties to check for user. If I sum up I see that sometimes you will have only `message` sometime "parameter" singular and `parameters` plural and the contained values doesn't always have the same type. Maybe simply adding them to the message is enough